docs/state.md

```diff
@@ -236,7 +236,47 @@ AppStore
     'tabs.switch-to-new-tabs': boolean, // true if newly opened tabs should be focused immediately
     'tabs.tabs-per-page': number // number of tabs per tab page
   },
-  sites: {
+  // TODO sort alphabetically
```

Comment:
re-sort on every sites mutation has been a source of serious perf issues, so as long as we don't do that it's okay.

i'm actually working on something similar for caching history and topsites. downside is the cache is available in the same process only.

it would be nice if there existed an Immutable.JS sorted map.